examples/prototypeApplication/advertConsent.json

```diff
@@ -497,27 +472,7 @@
       },
       "fee": {
         "calculated": [],
-        "payable": [],
-        "category": {
-          "one": [],
-          "two": [],
-          "three": [],
-          "four": [],
-          "five": [],
-          "sixAndSeven": [],
-          "eight": [],
-          "nine": [],
-          "ten": [],
-          "eleven": {
-            "one": []
-          },
-          "twelve": {
-            "one": [],
-            "two": []
-          },
-          "thirteen": [],
-          "fourteen": []
-        }
+        "payable": []
```

Comment:
Tidying up some application examples as I go to remove un-populated bits that are not relevant to this application type !